dsl/pace/dsl/future_stencil.py

```diff
@@ -50,7 +50,7 @@ class StencilTable(object, metaclass=Singleton):
 
     DONE_STATE: int = -1
     NONE_STATE: int = -2
-    MAX_SIZE: int = 200
+    MAX_SIZE: int = 220
```

Comment:
Only 20 more, physics is so simple! Will try to remember this as we add more stencils in physics.

Reply:
very true! the actual size is 2*MAX_SIZE + 1, so increased from 401 to 441. Yeah definitely, we can change this number as/when we need, and also the distributed compilation code may end up changing and not using this